lib/tasks/upgrade.rake

```diff
@@ -696,6 +708,160 @@ namespace :upgrade do
+        name: "fundref",
+        description: "Crossref Funder Registry (FundRef)",
+        active: true,
+        user_landing_url: "https://search.crossref.org/funding?q="
+      )
+    end
+    unless IdentifierScheme.where(name: "ror").any?
+      IdentifierScheme.create(
+        name: "ror",
+        description: "Research Organization Registry (ROR)",
+        active: true,
+        user_landing_url: "https://ror.org"
+      )
+    end
+  end
+
+  desc "update the Shibboleth scheme description"
+  task update_shibboleth_description: :environment do
+    scheme = IdentifierScheme.where(name: "shibboleth")
+    if scheme.any?
+      scheme.update(description: "Institutional Sign In (Shibboleth)")
+    end
+  end
+
+  desc "Contextualize the Identifier Schemes (e.g. which ones are for orgs, etc."
+  task contextualize_identifier_schemes: :environment do
+    IdentifierScheme.where(name: "shibboleth")
+                    .update_all(for_orgs: true, for_users: true, for_auth: true)
+
+    IdentifierScheme.where(name: "orcid")
+                    .update_all(for_users: true, for_auth: true)
+
+    IdentifierScheme.where(name: %w[ror fundref])
+                    .update_all(for_orgs: true)
+
+    IdentifierScheme.where(name: "doi")
+                    .update_all(for_plans: true)
+  end
+
+  desc "migrate the old user_identifiers over to the polymorphic identifiers table"
+  task convert_user_identifiers: :environment do
+    p "Transferring existing user_identifiers over to the identifiers table"
+    p "this may take in excess of 30 minutes depending on the size of your users table ..."
+    UserIdentifier.joins(:user, :identifier_scheme)
+                  .includes(:user, :identifier_scheme).all.each do |ui|
+
+      ui.user.identifiers << Identifier.new(
+        identifier_scheme: ui.identifier_scheme,
+        value: ui.identifier,
+        attrs: {}.to_json
+      )
+    end
+  end
+
+  desc "migrate the old org_identifiers over to the polymorphic identifiers table"
+  task convert_org_identifiers: :environment do
+    p "Transferring existing org_identifiers over to the identifiers table"
+    p "please wait ..."
+    OrgIdentifier.joins(:org, :identifier_scheme)
+                 .includes(:org, :identifier_scheme).all.each do |ui|
+
+      ui.org.identifiers << Identifier.new(
+        identifier_scheme: ui.identifier_scheme,
+        value: ui.identifier,
+        attrs: ui.attrs
+      )
+    end
+  end
+
+  desc "Sets the new managed flag for all existing Orgs to managed = true"
+  task default_orgs_to_managed: :environment do
+    Org.all.update_all(managed: true)
+  end
+
+  desc "Attempts to migrate other_organisation entries to Orgs"
+  task migrate_other_organisation_to_org: :environment do
+    names = User.all.where.not(other_organisation: nil)
+                .pluck(:other_organisation).uniq
+    names.each do |name|
+      # Skip any blank names
+      next unless name.gsub(/\s/, "").blank?
```

Comment:
Should this be either `if name.gsub(/\s/, "").blank?` or `unless name.gsub(/\s/, "").present?`

Reply:
yep. will patch in the `identifier-refactor` PR